src/main/java/com/cgdpd/library/service/BookService.java

```diff
@@ -24,4 +34,20 @@ public Book createBook(CreateBookRequestDTO requestDTO) {
         var createdBook = bookRepository.save(bookEntity);
         return bookMapper.mapToBook(createdBook);
     }
+
+    @Transactional(readOnly = true)
+    public PagedResponse<Book> getBooks(PaginationCriteria paginationCriteria,
+                                        SearchBookCriteria searchCriteria) {
+        Sort sort = paginationCriteria.sort()
```

Comment:
Use `var` instead of object type everywhere
Update ide settings
`editor -> inspections -> search for "Local Variable type can be omitted" -> change severity to warning`

<img width="956" alt="image" src="https://github.com/CGDPD/library/assets/43956748/d91bdc8a-d3e5-4795-a8f7-402054564b4e">